BaseLib/uniqueInsert.h

```diff
@@ -27,6 +27,20 @@ void uniquePushBack(Container& container, typename Container::value_type const&
         container.push_back(element);
 }
 
+//! Inserts the given \c key with the given \c value into the \c map if an entry with the
+//! given \c key does not yet exist; otherwise an \c error_message is printed and the
+//! program is aborted.
+template<typename Map, typename Key, typename Value>
+void insertMapIfKeyUniqueElseError(
```

Comment:
I'm missing something obvious, but what is the difference to the following `insertIfKeyUniqueElseError()` method?